crossbar/worker/container.py

```diff
@@ -322,33 +363,22 @@ def close_wrapper(orig, was_clean, code, reason):
 
                 return r
 
-            # FIXME: due to history, the following is currently the case:
-            # ITransportHandler.onClose is implemented directly on WampWebSocketClientProtocol,
-            # while with WampRawSocketClientProtocol, the ITransportHandler is implemented
-            # by the object living on proto._session
-            #
             if isinstance(proto, WampWebSocketClientProtocol):
```

Comment:
Is the statement in the comment now not true?

Reply:
Mm, I guess it is, but I took the comment in the context of an explanation of what needed fixing and why .. this code should fix the issue, and implicitly describes what was in the comment. I can leave the comment in if you like and just take out the "FIXME"?

Reply:
Okay, I'd like to keep the comment then (maybe with "NOTE" instead of "FIXME"...?) but this, I recall, being *very* confusing (and also the isinstance() is weird anyway, which is what it's explaining).